Packages/doc/installation.rst

```diff
@@ -162,11 +163,30 @@ Windows (with hardware support)
       -  ``Packages\MIES``
 
    -  In ``Igor Procedures`` a shortcut pointing to ``Packages\MIES_Include.ipf``
-
    -  In ``Igor Extensions (64-bit)`` a shortcut pointing to ``XOPs-IP9-64bit``
-
    -  In ``Igor Help Files`` a shortcut pointing to ``HelpFiles-IP9``
 
+-  ITC hardware:
+
+   -  Open a powershell console as administrator and execute the following scripts:
+
+      -  ``Packages\ITCXOP2\tools\Disable-ASLR-for-Igor64.ps1``
```

Comment:
@t-b should we also list the MIES menu option for running ASLR script? Should a menu option be added for the FixOffice365 script?

Reply:
@MichaelHuth 17175e23 (Installer update for ITC XOP compatibility, 2020-04-16) introduced the Fixoffice script. From the commit message I think this solves an issue for 32bit, or?

Reply:
Yes this resolves a 32 bit issue and we will remove it in #2364

Reply:
`should we also list the MIES menu option for running ASLR script`

@t-b, after rereading my message, I realized it wasn't clear. I'm suggesting we include instructions for disabling ASLR using the MIES menu option, along with directions for running the script manually.

Reply:
@timjarsky Fixed. I also don't mention the fix office script anymore.